mythril/interfaces/cli.py

```diff
@@ -53,138 +58,251 @@ def exit_with_error(format_, message):
+    :return: Parser which handles input
     """
-    parser.add_argument("solidity_file", nargs="*")
-
-    commands = parser.add_argument_group("commands")
-    commands.add_argument("-g", "--graph", help="generate a control flow graph")
-    commands.add_argument(
-        "-V",
-        "--version",
-        action="store_true",
-        help="print the Mythril version number and exit",
-    )
-    commands.add_argument(
-        "-x",
-        "--fire-lasers",
-        action="store_true",
-        help="detect vulnerabilities, use with -c, -a or solidity file(s)",
-    )
-    commands.add_argument(
-        "--truffle",
-        action="store_true",
-        help="analyze a truffle project (run from project dir)",
-    )
-    commands.add_argument(
-        "-d", "--disassemble", action="store_true", help="print disassembly"
-    )
-    commands.add_argument(
-        "-j",
-        "--statespace-json",
-        help="dumps the statespace json",
-        metavar="OUTPUT_FILE",
-    )
-
-    inputs = parser.add_argument_group("input arguments")
-    inputs.add_argument(
+    parser = ArgumentParser(add_help=False)
+    parser.add_argument(
         "-c",
         "--code",
         help='hex-encoded bytecode string ("6060604052...")',
         metavar="BYTECODE",
     )
-    inputs.add_argument(
+    parser.add_argument(
         "-f",
         "--codefile",
         help="file containing hex-encoded bytecode string",
         metavar="BYTECODEFILE",
         type=argparse.FileType("r"),
     )
-    inputs.add_argument(
+    parser.add_argument(
         "-a",
         "--address",
         help="pull contract from the blockchain",
         metavar="CONTRACT_ADDRESS",
     )
-    inputs.add_argument(
-        "-l",
-        "--dynld",
-        action="store_true",
-        help="auto-load dependencies from the blockchain",
-    )
-    inputs.add_argument(
-        "--no-onchain-storage-access",
-        action="store_true",
-        help="turns off getting the data from onchain contracts",
-    )
-    inputs.add_argument(
+    parser.add_argument(
         "--bin-runtime",
         action="store_true",
         help="Only when -c or -f is used. Consider the input bytecode as binary runtime code, default being the contract creation bytecode.",
     )
+    return parser
 
-    outputs = parser.add_argument_group("output formats")
-    outputs.add_argument(
+
+def get_output_parser() -> ArgumentParser:
+    """
+    Get parser which handles output
+    :return: Parser which handles output
+    """
+    parser = argparse.ArgumentParser(add_help=False)
+    parser.add_argument("--epic", action="store_true", help=argparse.SUPPRESS)
+    parser.add_argument(
         "-o",
         "--outform",
         choices=["text", "markdown", "json", "jsonv2"],
         default="text",
         help="report output format",
         metavar="<text/markdown/json/jsonv2>",
     )
-    outputs.add_argument(
+    parser.add_argument(
         "--verbose-report",
```

Comment:
The `-o/--outform` and `--verbose-report` arguments are displayed for disassemble, read-storage, leveldb-search, function-to-hash,  hash-to-address, and even version. They are only needed for `analyze`.

Reply:
`and even version` . json output stuff is being used in version.(but not the rest)